libraries/ESP8266WiFi/src/WiFiClientSecureBearSSL.h

```diff
@@ -121,48 +121,69 @@ class WiFiClientSecure : public WiFiClient {
     static bool probeMaxFragmentLength(const char *hostname, uint16_t port, uint16_t len);
     static bool probeMaxFragmentLength(const String& host, uint16_t port, uint16_t len);
 
-    bool setCACert(const uint8_t* pk, size_t size);
-    bool setCertificate(const uint8_t* pk, size_t size);
-    bool setPrivateKey(const uint8_t* pk, size_t size);
+    ////////////////////////////////////////////////////
+    // AxTLS API deprecated warnings to help upgrading
+
+#define AXTLS_DEPRECATED
+    #define xAXTLS_DEPRECATED \
```

Comment:
Looks like something's mixed up here.  Shouldn't everything be in the AXTLS_DEPRECATED macro and not xAXTLS...?

Reply:
fixed :)